src/config/constants.js

```diff
@@ -0,0 +1,48 @@
+import merge from 'lodash/merge';
+import packageJson from '../../package.json';
+
+// Default configurations applied to all environments
+const defaultConfigs = {
+    env: process.env.ENV || 'dev',
+    version: packageJson.version,
+};
+
+// Environment specific overrides
+const environmentConfigs = {
+    dev: {
+        cognito: {
+            userPoolId: process.env.USER_POOL_ID || 'us-east-1_HetaxzWw5',
+            ISS: process.env.USER_POOL_ID || 'https://cognito-idp.us-east-1.amazonaws.com/us-east-1_HetaxzWw5/',
+        },
+        db: {
+            host: process.env.USER_POOL_ID || '52.15.81.107',
```

Comment:
@singhabahu  @SavidyaSathyani  Just want to know why we are putting this way "process.env.USER_POOL_ID || '52.15.81.107'". Is process.env.USER_POOL_ID default value or is there any convention?